### PR TITLE
void transaction

### PR DESCRIPTION
**Pull request by jtgrimes**
Payflow has a VOID transaction which is implemented in this PR

https://developer.paypal.com/docs/classic/payflow/integration-guide/#submitting-void-transactions

### CODE REVIEW

**Comment by jtgrimes** [comment]
Any word on this PR? I'm happy to do anything that needs doing to move this along.

File: src/Message/VoidRequest.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Omnipay\Payflow\Message;
+
+/**
+ * Payflow Void Request
+ */
+class VoidRequest extends AuthorizeRequest
+{
+    protected $action = 'V';
```
Comment (kayladnls) [inline src/Message/VoidRequest.php]: doc bloc here, please, and a space between property and the next function.

**Comment by kayladnls** [comment]
one tiny comment and then I'll merge away. :)

**Comment by kayladnls** [comment]
:+1: :confetti_ball: :confetti_ball: Thanks!

**Pull request merged**